src/Network/Network/ExpressRouteCircuit/Peering/AddAzureExpressRouteCircuitPeeringConfigCommand.cs

```diff
@@ -69,7 +69,7 @@ public override void Execute()
                 peering.SharedKey = this.SharedKey;
             }
 
-            if (this.PeerAddressType == IPv6)
+            if (string.Equals(this.PeerAddressType, "IPv6", StringComparison.OrdinalIgnoreCase))
```

Comment:
As the logic of checking `PeerAddressType` is very common. I suggest to create some util function for these logic.

Reply:
Follow up logic beyond the check is different in different files. Not sure what to do with utility function.

Are you suggesting to define a bool util function which implements
(string.Equals(this.PeerAddressType, "IPv6", StringComparison.OrdinalIgnoreCase))

Reply:
Yes. Also for `IPv4`.

Reply:
done.